src/ReactiveUI.AndroidX/ReactiveRecyclerViewViewHolder.cs

```diff
@@ -88,47 +92,61 @@ protected ReactiveRecyclerViewViewHolder(View view)
          h => view.LongClick -= h);
     }
 
+    /// <inheritdoc/>
+    public event PropertyChangedEventHandler? PropertyChanged;
+
     /// <inheritdoc/>
     public event PropertyChangingEventHandler? PropertyChanging;
 
     /// <inheritdoc/>
-    public event PropertyChangedEventHandler? PropertyChanged;
+    public IObservable<Unit> Activated => _activated.AsObservable();
+
+    /// <inheritdoc/>
+    [IgnoreDataMember]
+    public IObservable<IReactivePropertyChangedEventArgs<ReactiveRecyclerViewViewHolder<TViewModel>>> Changed => this.GetChangedObservable();
+
+    /// <inheritdoc/>
+    [IgnoreDataMember]
+    public IObservable<IReactivePropertyChangedEventArgs<ReactiveRecyclerViewViewHolder<TViewModel>>> Changing => this.GetChangingObservable();
+
+    /// <inheritdoc/>
+    public IObservable<Unit> Deactivated => _deactivated.AsObservable();
 
     /// <summary>
-    /// Gets an observable that signals that this ViewHolder has been selected.
+    /// Gets an observable that signals that this ViewHolder has been long-clicked.
     ///
     /// The <see cref="int"/> is the position of this ViewHolder in the <see cref="RecyclerView"/>
     /// and corresponds to the <see cref="RecyclerView.ViewHolder.AdapterPosition"/> property.
     /// </summary>
-    public IObservable<int> Selected { get; }
+    public IObservable<int> LongClicked { get; }
```

Comment:
Don't really want all these random changes where properties are located, thought it was a breaking change for a bit.

Reply:
I also just spotted this, sorry I had my auto format and sort on which caused the rearrangements, I will revert the few files it formatted and reapply the fixes